background/addons-list.rst

```diff
@@ -0,0 +1,66 @@
+.. _django-addons-list:
+
+Django addons list
+==================
+
+Divio provides a number of Django addon applications for convenience.
+
+*Support* refers to Divio's commitment to maintain an addon, which could include bug-fixes and continued development of
+features. It does not refer to technical support for end users.
+
+Fully supported
+---------------
+
+Addons that Divio takes responsibility for, and that can be expected to be up-to-date with current LTS versions of
+Django.
+
+* `Django Filer <https://github.com/django-cms/django-filer>`_
+* `django CMS Link <https://github.com/django-cms/djangocms-link>`_
+* `django CMS Picture <https://github.com/django-cms/djangocms-picture>`_
+* `django CMS File <https://github.com/django-cms/djangocms-file>`_
+* `django CMS Snippet <https://github.com/django-cms/djangocms-snippet>`_
+* `django CMS Style <https://github.com/django-cms/djangocms-style>`_
+* `django CMS Audio <https://github.com/django-cms/djangocms-audio>`_
+* `django CMS Video <https://github.com/django-cms/djangocms-video>`_
+* `django CMS Icon <https://github.com/django-cms/djangocms-icon>`_
+* `django CMS Attributes Field <https://github.com/django-cms/djangocms-attributes-field>`_
+* `django CMS History <https://github.com/django-cms/djangocms-history>`_
+* `django CMS Modules <https://github.com/django-cms/djangocms-modules>`_
+* `django CMS Transfer <https://github.com/django-cms/djangocms-transfer>`_
+* `Aldryn Django <https://github.com/divio/aldryn-django>`_
+* `Aldryn django CMS <https://github.com/divio/aldryn-django-cms>`_
+* `Aldryn Addons <https://github.com/divio/aldryn-addons>`_
+* `Aldryn SSO <https://github.com/divio/aldryn-sso>`_
+* `Aldryn Snake <https://github.com/divio/aldryn-snake>`_
+
+Limited support
```

Comment:
In the DHB, there should be two blank lines between body content and the next header. Can you add this please?

Reply:
Done